.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "runtime/integration-tests/submodules/liquidity-pools"]
+	path = runtime/integration-tests/submodules/liquidity-pools
+	url = git@github.com:centrifuge/liquidity-pools.git
+	branch = feat/local-router
```

Comment:
Eventually, we want to use the `main` branch here right?

Reply:
Feature branch for the local router we need for routing to local evm. Will be master once this is merged

Reply:
Update: Main branch cannot be used as the `LocalRouter` feat is required. We have been using the `centrifuge-chain/release-v1.0` branch: https://github.com/centrifuge/liquidity-pools/pull/260